x-pack/plugins/lens/public/types.ts

```diff
@@ -470,6 +470,7 @@ export type VisualizationDimensionGroupConfig = SharedDimensionProps & {
   supportsMoreColumns: boolean;
   /** If required, a warning will appear if accessors are empty */
   required?: boolean;
+  minDimensions?: number;
```

Comment:
seeing the comment from Joe, this will probably change? If for some reason it would still be here(for example, we'd decide to change the messaging instead from `Required dimension` to `Required 2 dimensions`) , I wonder if it would make sense to combine it with `required` as `requiredMinDimensionCount`... Or something shorter 😅

Reply:
agree @mbondyra - the stripping of date histogram seems to make the most sense only from XY --> Partition chart type (and not between the partition chart types)